config/dictionaries/cloud.json

```
@@ -0,0 +1,56 @@
+[
+  {
+    "id": "aws",
+    "name": "AWS",
+    "shortName": "AWS",
+    "fullName": "AWS",
+    "platformName": "AWS IoT",
+    "platformUrl": "https://aws.amazon.com/iot",
+    "fullPlatformName": "AWS IoT",
+    "iotCoreName": "IoT Core",
+    "iotCoreUrl": "https://aws.amazon.com/iot-core",
+    "credentialType": "public key certificate",
+    "provisionRepoName": "aws-iot-provision",
+    "provisionRepoUrl": "https://github.com/balena-io-examples/aws-iot-provision",
+    "cloudFunctionName": "Lambda function",
+    "cloudFunctionFullNameStrong": "**AWS Lambda (cloud) function**",
+    "accountSetupLink": "https://github.com/balena-io-examples/aws-iot-provision#aws-setup",
+    "toolsSetupLink": "https://github.com/balena-io-examples/aws-iot-provision#tools-setup"
```

```suggestion
    "toolsSetupLink": "https://github.com/balena-io-examples/aws-iot-provision#development-setup"
```

If you modify this heading to my suggestion then you wouldn't need these properties

---

I was trying to make a distinction between generic platform setup and tools setup. Let's discuss.

---

We decided to leave as is.